docs/src/terminology.md

```diff
@@ -217,9 +217,16 @@ A 64-byte ed25519 signature of R (32-bytes) and S (32-bytes). With the requireme
 This requirement ensures no signature malleability. Each transaction must have at least one signature for [fee account](terminology#fee-account).
 Thus, the first signature in transaction can be treated as [transacton id](terminology.md#transaction-id)
 
+## skipped slot
+
+A past [slot](terminology.md#slot) where its scheduled leader didn't produce a [block](terminology.md#block) due to off-line or the [fork](terminology.md#fork) containing the slot was abandoned over a better alternative at the time by the cluster consensus. Being _skipped_ means that blocks at subsequent slots omits to include the skipped slots as ancestors, possibly creating an after-the-fact short cluster stall of transaction processing, oldest `recent_blockhash` expiration, [block height](terminology#block-height) increment, etc.
+
+By definition, being skipped can be determined only after being older than the latest [rooted](terminology.md#root) (thus not-skipped) slot.
+
 ## slot
 
-The period of time for which a [leader](terminology.md#leader) ingests transactions and produces a [block](terminology.md#block).
+A numbered non-overlapping slice of logical time, ordered sequentially with roughly equal passage of real wall-clock utilizing [PoH](terminology.md#proof-of-history).
```

Comment:
hehe, sorry for self-bragging but, I like this under-20-word crisp definition for slot. Previously, I think this entry lacked this crucial contextual information, immediately talking about something leader block production... :)

Reply:
Hmm, I actually like the previous definition. How about leaving the original sentence, and then adding the contextual detail?

Reply:
@CriesofCarrots Got it. Thanks for honest opinion. :) How does this look now? I still tweaked the original definition a bit.